src/plugin/timeline.js

```diff
@@ -16,7 +16,9 @@
  * performance for large files
  * @property {string} fontFamily='Arial'
  * @property {number} fontSize=10 Font size of labels in pixels
- * @property {function} formatTimeCallback=→00:00
+ * @property {number} startSeconds=0 Start time in second for begining timeline rule
+ * @property {string} formatTime='MM:SS' default time format for timeline rule, can be change to'HH:MM:SS'
```

Comment:
2 typos: `change` should be `changed` and missing space after `to`

Reply:
OK